devtools/php-cs-fixer/install-php-cs-fixer.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/sh
```
No bash features used, so it is not required.

Reply:
see: https://github.com/bareos/bareos/pull/1403/commits/b0b7dbee24106eb8d91116aa3905d61bf75f344a